site/shared/src/main/scala/shared/User.scala

```diff
@@ -23,3 +23,26 @@ case class UserProgress(
   args:         Option[String],
   succeeded:    Boolean
 )
+
+case class OverallUserProgressItem(
+  libraryName: String,
+  sections:    Int,
+  completed:   Int
+)
```

Comment:
Sure, I'd need to know:

* Where can I fetch that information.
* The type (String??)